cmd/stack_rename.go

```diff
@@ -41,7 +41,11 @@ func newStackRenameCmd() *cobra.Command {
 			"Note: Because renaming a stack will change the value of `getStack()` inside a Pulumi program, if this\n" +
 			"name is used as part of a resource's name, the next `pulumi up` will want to delete the old resource and\n" +
 			"create a new copy. For now, if you don't want these changes to be applied, you should rename your stack\n" +
-			"back to its previous name.",
+			"back to its previous name." +
+			"\n" +
+			"You can also rename the stack's project by passing a fully-qualified stack name as well. For example:\n" +
```

Comment:
Does it make sense to also mention here you cannot update the organization?

Reply:
I'm inclined to leave as-is not mentioning the "cannot transfer ownership via rename", as it is probably just going to confuse readers. (Who are interested in understanding `stack rename` and not looking for `stack transfer` or some other command.)

The error message a user gets if they do try to "transfer via rename" should be sufficient.